scale-tests/deploy-dispatchers.py

```diff
@@ -236,20 +228,24 @@ def deploy_dispatchers(
     """
     Deploy the required number of dispatchers and store their information to a text file.
     """
-
-    with open(output_file, "w") as outfile:
-        dispatchers = []
-        for i in range(num_dispatchers):
-            dispatcher_settings = install_package("spark", service_name_base, i,
-                                                linux_user,
-                                                lambda x: 0,
-                                                None,
-                                                options,
-                                                quota_options)
-            dispatchers.append(dispatcher_settings)
-            outfile.write("{},{},{}\n".format(dispatcher_settings["service"]["name"],
-                                              dispatcher_settings["roles"]["drivers"],
-                                              dispatcher_settings["roles"]["executors"]))
+    def deploy_dispatcher(i):
```

Comment:
We could add type annotations, and I'm on the fence about whether we need to use a different parameter name than `i`. (`index` perhaps?)